server/src/main/resources/org/cloudfoundry/identity/uaa/db/postgresql/V3_10_4__Add_Index_On_Identity_Zone_To_Revocable_Tokens.sql

```diff
@@ -0,0 +1,2 @@
+-- create an index to match the query
+CREATE INDEX revocable_tokens_zone_id ON revocable_tokens(LOWER(identity_zone_id));
```

Comment:
Thanks for pointing it out. 
Postgresql flyway script has been changed to remove the `lower` function